.travis.yml

```diff
@@ -54,17 +45,15 @@ install:
   - pip freeze
 
 script:
-  # For Linux run only the tests on 3.5 and the full checker elsewhere, all need the "X Virtual Framebuffer"
-  - if [ "$TRAVIS_OS_NAME" = "linux" ] && [ "$TRAVIS_PYTHON_VERSION" = "3.5" ]; then xvfb-run make coverage; fi
-  - if [ "$TRAVIS_OS_NAME" = "linux" ] && [ "$TRAVIS_PYTHON_VERSION" != "3.5" ]; then xvfb-run make check; fi
```

Comment:
The tests are un as part of `make macos` below, but removing these two lines basically removes running the tests on Linux, so I think they need to stay.